### DISCUSSION TITLE
Encounter  a problem with "China Open Source Map"

### DISCUSSION THREAD

**Discussion by ufo5260987423** [kaiyuanshe/kaiyuanshe.github.io#103] [Q&A] [answered]
I'd like to register my repo[scheme-langserver](https://github.com/ufo5260987423/scheme-langserver), but there are no gentle domain for [language server protocol](https://microsoft.github.io/language-server-protocol/) implementation. 

Would anyone help me? Or this community would extend the list?

**Comment by TechQuery** [comment, marked as answer]
I have created a new tag `编程语言` for programming languages, I'm glad to have your awesome project registered!

**Reply by ufo5260987423** [reply]
Thank you for your response!
But  scheme-langserver is a personal repo and don't have relations with any organizations. 
What's the best practice for repo like mine? Would there be any instruction?

**Reply by TechQuery** [reply]
This map accepts any kinds of open source projects or organizations. So open  matters, organization does not.
